.infra/workers.ts

```diff
@@ -215,4 +215,8 @@ export const workers: Worker[] = [
     topic: 'analytics-api.v1.experiment-allocated',
     subscription: 'api.experiment-allocated',
   },
+  {
+    topic: 'api.v1.member-joined-source',
+    subscription: 'api.member-joined-source-mailing',
+  },
```

Comment:
Added new subscription here, better then cram up this logic into existing that is dedicated to notifications?